### PR TITLE
[election-api-js]: Clarify test comments

### PR DESCRIPTION
**Pull request by paulbrimicombe**
## What?

Clarify what assertions we are expecting in the JS API tests and use natural language rather than code symbols.

## Why?

The use of `=` and `==` in the testing comments has caused some confusion for some candidates

### CODE REVIEW

**Review by nicholasgriffintn — APPROVED** [review]
This is much better in my opinion, doesn't prescribe an answer and gives pointers in plain English.

**Comment by CameronWatt98** [comment]
I also like this. I think not writing "winner = none" will lead candidates to possibly implement cleaner solutions that don't involve using a hardcoded string of "none".

**Review by TimPerry — APPROVED** [review]
Yeah, agree with others

**Review by billsheikh1 — APPROVED** [review]
Really good. The `=` and `==` caused so many questions during interviews
